alphagov/paas-cf
# [#183781224] Improve reliability of autoscaler acceptance tests in `dev`

What
----

2 real changes here. Details are in the individual commit messages. I went a little past the scope of the story and fixed up all the acceptance tests so they better adhered to the recommentations in [ginkgo's documentation](https://onsi.github.io/ginkgo/#recommended-continuous-integration-configuration).

How to review
-------------

- Deploy [alphagov/paas-bootstrap#546](https://github.com/alphagov/paas-bootstrap/pull/546) to a dev env
- Make the pipelines from this branch and ensure the behaviour matches the behaviour described in the commit messages.

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
